polyfill/lib/regex.mjs

```diff
@@ -7,7 +7,7 @@ export const calendarID = new RegExp(`(?:${calComponent.source}(?:-${calComponen
 const yearpart = /(?:[+\u2212-]\d{6}|\d{4})/;
 export const datesplit = new RegExp(`(${yearpart.source})(?:-(\\d{2})-(\\d{2})|(\\d{2})(\\d{2}))`);
 const timesplit = /(\d{2})(?::(\d{2})(?::(\d{2})(?:[.,](\d{1,9}))?)?|(\d{2})(?:(\d{2})(?:[.,](\d{1,9}))?)?)?/;
-export const offset = /([+\u2212-])([0-2][0-9])(?::?([0-5][0-9])(?::?([0-5][0-9])(?:[.,](\d{1,9}))?)?)?/;
+export const offset = /([+\u2212-])([0-1][0-9]|[2][0-3])(?::?([0-5][0-9])(?::?([0-5][0-9])(?:[.,](\d{1,9}))?)?)?/;
```

Comment:
Small improvement: `[0-1]` can be simplified to `[01]` (since no range is necessary) and `[2]` can be simplified to `2` (since it is only one character, not a choice.)

Reply:
Thanks, makes sense, I'll make the change.